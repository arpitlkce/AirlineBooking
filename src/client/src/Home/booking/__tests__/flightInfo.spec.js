import React from 'react';
import renderer from 'react-test-renderer';
import FlightInfo from '../flightInfo';

describe('FlightInfo Component', () => {
    it('should render correctly in  mode', () => {
        let props = {
            'duration': '4:00',
            'price': '1240,20',
            'departureAirportCode': 'BLR',
            'arrivalAirportCode': 'LKO',
            'flightNumber': 'IN-220',
            'flightName': 'Indigo',
            'departureTime': '12:35',
            'departureAirport': 'Bangalore',
            'arrivalTime': '14:35',
            'arrivalAirport': 'Lucknow'
        };
        const component = renderer.create(<FlightInfo item={props} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
