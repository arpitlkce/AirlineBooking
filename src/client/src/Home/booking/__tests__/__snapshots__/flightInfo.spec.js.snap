// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlightInfo Component should render correctly in  mode 1`] = `
<div
  className="jumbotron jumbotron-fluid"
>
  <div
    className="container-fluid"
  >
    <div
      className="lead row"
    >
      <div
        className="col"
      >
        <label
          className=""
        >
          Flight Details 
        </label>
      </div>
      <div
        className="col"
      >
        <label
          className=""
        >
          Departure Details
        </label>
      </div>
      <div
        className="col"
      >
        <label
          className=""
        >
          Arrival Details
        </label>
      </div>
      <div
        className="col"
      >
        <label
          className=""
        >
          Duration
        </label>
      </div>
      <div
        className="col"
      >
        <label
          className=""
        >
          Price
        </label>
      </div>
    </div>
    <div
      className="small lead row"
    >
      <div
        className="col"
      >
        <div>
          Indigo
        </div>
        <div>
          IN-220
        </div>
      </div>
      <div
        className="col"
      >
        <div>
          12:35
        </div>
        <div>
          BLR
        </div>
        <div>
          Bangalore
        </div>
      </div>
      <div
        className="col"
      >
        <div>
          14:35
        </div>
        <div>
          LKO
        </div>
        <div>
          Lucknow
        </div>
      </div>
      <div
        className="col"
      >
        <div>
          4:00
           Hrs
        </div>
      </div>
      <div
        className="col"
      >
        <div>
          $ 
          1240,20
        </div>
      </div>
    </div>
  </div>
</div>
`;
